server/startup/removeOfflineGuests.js

```diff
@@ -0,0 +1,14 @@
+Meteor.startup(() => {
+	let query = {
```

Comment:
The `query` isn't being assigned a value, so I think this should be `const` instead of `let`.